pkg/descheduler/descheduler.go

```diff
@@ -138,7 +140,29 @@ func cachedClient(
 
 	for _, item := range namespaces {
 		if _, err := fakeClient.CoreV1().Namespaces().Create(context.TODO(), item, metav1.CreateOptions{}); err != nil {
-			return nil, fmt.Errorf("unable to copy node: %v", err)
+			return nil, fmt.Errorf("unable to copy namespace: %v", err)
+		}
+	}
+
+	persistentVolumes, err := pvInformer.Lister().List(labels.Everything())
```

Comment:
Can these also be guarded by conditions for when it's actually needed?

Reply:
I've only added one parameter, `ignoreLocalPvcPods`, and in either case (true for false) it needs to be able check the PVC and the PV to determine if it is indeed a "local" (or hostPath) type PV.  We could break it up into two config options, one for checking and one for deleting, but that might get (more) confusing?